docs_manual/source/metadata_advanced.rst

```diff
@@ -216,3 +218,40 @@ Protein Expression Profiling / Mass Cytometry Plugin
     * *Raw Data Files* and *Derived Data Files* are linked to ``{Assay Name}``
 - Used with measurement type / technology type
     * protein expression profiling / mass cytometry
+
+Universal Assay Plugin
+----------------------
+
+This plugin enables the user to define row-specific and inline links to
+iRODS collections via comments in the ``STUDY ASSAYS`` section of the ISA-Tab
+investigation file.
```

Comment:
Maybe add further clarification like "This plugin can be used with any assay measurement and technology configuration, as long as the desired linking is given in ISA-Tab comments"?

By the way, this description of the plugin is a good idea. I'll have to add similar short ones for the others as well.